docs/ReleaseNotes.md

```diff
@@ -1,6 +1,42 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.0
+Acuminator 2.0 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Key Changes
+Acuminator 2.0 includes the following key changes:
+
+ - Acuminator no longer supports Visual Studio 2015. 
```

Comment:
@k-s-popova maybe we need to tell more about the transfer to new Roslyn and new VS SDK methods. While not being a separate feature itself these changes should improve UX and UI response time and extension's load time.

Reply:
fixed